packages/core/types/index.js

```diff
@@ -299,6 +302,42 @@ export interface Asset extends BaseAsset {
   +stats: Stats;
 }
 
+// TODO: figure out naming
+export interface IConfig {
```

Comment:
I'm not sure all of this should be a public type. With our other types, we have an internal version with fields like the below, and a public one with the public interface. It think we should aim to match for config as well.

As for the interface name, we've generally named it without the prefix, so just `Config` instead of `IConfig`.

Reply:
Yeah, again was kind of neglecting internal/public differences.

> As for the interface name, we've generally named it without the prefix, so just Config instead of IConfig.

Ah yeah, thought we had something like IDependency in the types package before, but it looks like I either have a bad memory or we changed it. I'll update the name :)